src/plugins/imagery/components/ImageryView.vue

```diff
@@ -66,6 +74,8 @@
             :style="getVisibleLayerStyles(layer)"
           ></div>
           <img
+            role="img"
```

Comment:
`<img>` elements implicitly have an aria role of `"img"` so we can just remove this